tensorflow/contrib/learn/python/learn/tests/summary_writer_test.py

```diff
@@ -0,0 +1,43 @@
+# pylint: disable=g-bad-file-header
```

Comment:
Thanks for adding a test! Please move it next to the implementation, i.e. to `tensorflow/python/summary/summary_writer_test.py`.